packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -389,6 +403,7 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
 
   /// Returns a widget displaying the map view - deprecated, use
   /// [buildViewWithConfiguration] instead.
```

Comment:
This deprecation-ish comment should be removed; it was what we did when we couldn't use actual `@Deprecation`. So the comment should now just be `/// Returns a widget displaying the map view.`